test/JDBC/expected/BABEL-3478-vu-verify.out

```diff
@@ -0,0 +1,99 @@
+SELECT * FROM BABEL_3478_t1 ORDER BY LastName ASC;
```

Comment:
you have two "Doe"s, sorting order of these two rows are not guaranteed. 
Rule of thumb: we are adding "order by " to create consistent result order. 
For your case, consistent order can be order by "first and last name"